jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/SingleProjectIntegrationTest.java

```diff
@@ -157,11 +167,14 @@ private static void buildAndRunComplex(
     assertDockerInspect(imageReference);
     String history = new Command("docker", "history", imageReference).run();
     Assert.assertThat(history, CoreMatchers.containsString("jib-gradle-plugin"));
+
+    String output = new Command("docker", "run", "--rm", imageReference).run();
     Assert.assertEquals(
         "Hello, world. An argument.\n1970-01-01T00:00:01Z\nrwxr-xr-x\nrwxrwxrwx\nfoo\ncat\n"
             + "1970-01-01T00:00:01Z\n1970-01-01T00:00:01Z\n"
             + "-Xms512m\n-Xdebug\nenvvalue1\nenvvalue2\n",
-        new Command("docker", "run", "--rm", imageReference).run());
+        output);
+    return output;
```

Comment:
Is returning `output` ever useful? Because the above assertion already checks that the output is always some fixed string. There cannot be any variation.

Reply:
It's used in `testBuild_complex` to make sure the output is the same when it's run using the digest and image id. I wanted to avoid copy-pasting the long string.

Reply:
Yeah, I saw it's being used, but what I saying is that whenever `buildAndRunComplex()` is called, the output is some fixed string which is verified inside `buildAndRunComplex()`. The returned output will always be same no matter how many you call `buildAndRunComplex()` differently, so comparing the two output strings seems redundant.

Reply:
Maybe I'm not following, but the reason for returning the output (even though yes, it should never change) is because building with the digest/running with the id should return identical output to the original build; it doesn't matter exactly what that output is, just that it's the same. Would it be better style to assert the output outside of `buildAndRunComplex()`?

Reply:
Ah, sorry. I misunderstood.